src/connectdlg.cpp

```diff
@@ -220,15 +211,16 @@ void CConnectDlg::RequestServerList()
 
     // update list combo box (disable events to avoid a signal)
```

Comment:
This needs a bit more explanation now.

There are two fields being added together and it's not clear how the two relate.  I can see an address type being turned into an integer - fine, make sense as an index.  I can see a "custom directory index" - but no explanation what that means, so I guess it's an index into a list of custom directory servers.  My initial thought is that both could be non-zero.  Or both zero and be valid.

So I then think I must be missing something.  It's that something that needs explaining here.